Content.Server/GameObjects/Components/Items/Storage/ServerStorageComponent.cs

```diff
@@ -494,5 +507,14 @@ void IExAct.OnExplosion(ExplosionEventArgs eventArgs)
                 }
             }
         }
+        public void PlayRustleEffect()
+        {
+            if (!string.IsNullOrWhiteSpace(_soundCollectionName))
+            {
+                var soundCollection = _prototypeManager.Index<SoundCollectionPrototype>(_soundCollectionName);
+                var file = _random.Pick(soundCollection.PickFiles);
```

Comment:
Instead of doing this manually, use `AudioHelpers.GetRandomFileFromSoundCollection()`